MetaU Capstone Project/API/SpotifyManager.h

```diff
@@ -6,10 +6,24 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "SpotifyiOS/SpotifyiOS.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface SpotifyManager : NSObject
+@interface SpotifyManager : NSObject <SPTAppRemoteDelegate, SPTSessionManagerDelegate>
+
++ (id)shared;
```

Comment:
Maybe you should specify the class type here.
Instead of `id`, use `SpotifyManager *` ?

So it would be:
`- (SpotifyManager *)shared;`